tests/ClaimTest.cpp

```diff
@@ -59,7 +59,7 @@ TEST(ClaimTest, AudienceAsSet) {
 TEST(ClaimTest, SetAudienceAsSet) {
 	auto token = jwt::create()
 		.set_type("JWT")
-		.set_audience({picojson::value("test"), picojson::value("test2")})
+		.set_audience({{picojson::value("test"), picojson::value("test2")}})
```

Comment:
What is the purpose of this change?

Reply:
I was getting the following error when compiling with (clang-1100.0.33.17)

`error: call to member function 'set_audience' is ambiguous
                .set_audience({picojson::value("test"), picojson::value("test2")})
                ~^~~~~~~~~~~~
.../jwt-cpp/include/jwt-cpp/jwt.h:2263:12: note: candidate function
                builder& set_audience(typename json_traits::array_type a) { return set_payload_claim("aud", typename json_traits::value_type(a)); }
                         ^
.../jwt-cpp/include/jwt-cpp/jwt.h:2269:12: note: candidate function
                builder& set_audience(typename json_traits::string_type aud) { return set_payload_claim("aud", typename json_traits::value_type(aud)); }`

maybe a clang thing, I don't see this on gcc 9.2.0.

Reply:
Hmmm interesting!